tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml

```diff
@@ -35,6 +40,26 @@ jobs:
       DockerBuildArgs: "--build-arg BUILD_UID=$( id -u )"
       Repository: onnxruntimecpubuild
 
+  - script: |
+      sudo apt-get install ccache -y
+      echo "##vso[task.prependpath]/usr/lib/ccache"
+    displayName: Install ccache and update PATH to use linked versions of gcc, cc, etc
```

Comment:
We run the build in containers. Why do you need to install ccache on the host OS?

Reply:
Good catch. I'll try to remove it.